### DISCUSSION TITLE
Lazer UI and QoL

### DISCUSSION THREAD

**Discussion by VV-LX** [ppy/osu#28839] [Ideas] [answered]
If osu's Stable build is like a teenage Ship of Theseus, lazer is a yacht designed by a grad student. One's clearly more contemporary, but it's still behind its industry peers.

### Friends List / F9

- When in Friends List, there should be an option to search for names among Friends. This is helpful when you can only partially remember a name or when a name was changed, and is much faster than sorting by username or searching among all players 

- Card View doesn’t need to be as compact as it is. A 3x3 like with “currently online” or even 2x3 would be better, letting you observe and process the decorations your friends make to their profiles

- The Supporter Heart is far too big and distracting and can clash with whatever backgrounds users have uploaded

- The Online/Offline Status Ring and the Online/Offline text next to it are just redundant and too big (A dot at the bottom of the profile like how Discord is should be fine)

### In-game Tournament Page 
An integrated, streamlined sign-up for competitive events. Having tournament listings held on the Forum is clunky and adds another barrier limiting the accessibility of competitive osu. Take League of Legend’s Clash page/Fortnite’s Tournament Page for example. 

Having sign-ups handled on-site would also make shoddy management less possible.

### Profile Customization
Pinned scores are a nice consideration for letting players choose which scores they want people to focus on, likely made with the intention of drawing away attention from the biggest pp number This isn’t enough, however. 
Unless you’ve played the map yourself, you’ll still look at indicators like acc and pp to judge the play, or ignore it altogether. 
If you have played the map, unless the score is clearly impressive, it’ll still be judged using the above two indicators. 

- What if players could show Highlights of moments from that score directly alongside it which could be watched

- Allow more integration with other apps like Discord, Steam and osu-specific clients. I’m unsure how every Beatmap Nominator manages their Queues, but it would be more useful if it was as prominently visible as their Rank and could be interacted with on-site/in-game

- Having each BN’s accepted queue be visible on their page would also be useful

- Let players create/post on a user feed, ordered in a timeline fashion like Twitter/X or Instagram. Recency is key, and they could display their "Playlists" of favorite beatmap groupings, new scores, thoughts, or any other achievements (Replacing "Latest Medals").

**Comment by bdach** [comment]
> If osu's Stable build is like a teenage Ship of Theseus, lazer is a yacht designed by a grad student. One's clearly more contemporary, but it's still behind its industry peers.

maybe skip this part next time? not particularly great to see feedback frontloaded by an insult

**Reply by VV-LX** [reply]
True, my bad. Lazer has room for features that feel much more prominent when missing.

**Reply by peppy** [reply, marked as answer]
The funny part is the feedback provided is all very tame stuff which is like "sure, we'll be adding that when we get to it". So I'm going to take the initial sentence as a compliment on both fronts.
